<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4" rowspan="4">
     <widget class="ItemView" name="collapsedItemViewer" native="true"/>
    </item>
    <item row="3" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2" rowspan="4">
     <widget class="AffixRollsViewer" name="affixRollsViewer" native="true"/>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="ModifierViewer" name="modifierViewer" native="true"/>
    </item>
    <item row="0" column="0" rowspan="4">
     <widget class="BaseItemViewer" name="baseItemViewer" native="true"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Random item</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pbFloatPhysical">
      <property name="text">
       <string>Flat physical</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pbPctPhysical">
      <property name="text">
       <string>Pct physical</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1190</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseItemViewer</class>
   <extends>QWidget</extends>
   <header>baseitemviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModifierViewer</class>
   <extends>QWidget</extends>
   <header>modifierviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AffixRollsViewer</class>
   <extends>QWidget</extends>
   <header>affixrollsviewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemView</class>
   <extends>QWidget</extends>
   <header>itemview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
